doc/NPCs.md

```diff
@@ -1261,12 +1261,13 @@ Example | Description
 `"u_val": "mana_max"` | Max mana. Can be read but not written to.
 `"u_val": "hunger"` | Current perceived hunger. Can be read but not written to.
 `"u_val": "thirst"` | Current thirst.
+`"u_val": "instant_thirst"` | Current thirst.
```

Comment:
what the difference?

Reply:
correct terminology so that it gets the right variable.